crates/evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -511,6 +517,16 @@ fn sleep(milliseconds: &U256) -> Result {
     Ok(Default::default())
 }
 
+/// Returns the time since unix epoch in milliseconds
+fn time() -> Result {
+    let sys_time = SystemTime::now();
+    let difference =
+        sys_time.duration_since(UNIX_EPOCH).expect("Failed getting timestamp in time cheatcode");
+    let millis = difference.as_millis();
+    let as_u8_arr: [u8; 32] = rU256::from(millis).to_be_bytes();
+    Ok(Bytes::from(as_u8_arr))
```

Comment:
we need to encode this, types are currently a bit horrible until we completely transitioned to alloy types we need to use:
`Ok(rU256::from(millis).to_ethers().encode().into())`